src/gui/MItem_menus.cpp

```diff
@@ -402,17 +402,37 @@ MI_LOAD_SETTINGS::MI_LOAD_SETTINGS()
     : WI_LABEL_t(_(label), nullptr, is_enabled_t::yes, is_hidden_t::no) {
 }
 
+struct SettingsLoadState {
+    bool is_network;
+    bool is_connect;
+};
+
 void MI_LOAD_SETTINGS::click(IWindowMenu & /*window_menu*/) {
-    // FIXME: Some error handling/reporting
-    // TODO: Loading other things than just network
+    SettingsLoadState settings_loaded = SettingsLoadState { .is_network = true, .is_connect = true };
+
     if (netdev_load_ini_to_eeprom()) {
         notify_reconfigure();
+    } else {
+        settings_loaded.is_network = false;
     }
 
-// FIXME: Error handling
 #if BUDDY_ENABLE_CONNECT()
-    connect_client::MarlinPrinter::load_cfg_from_ini();
+    if (!connect_client::MarlinPrinter::load_cfg_from_ini()) {
+        settings_loaded.is_connect = false;
+    }
+#endif
+    if (settings_loaded.is_connect & settings_loaded.is_network) {
+        MsgBoxInfo(_("All settings loaded."), Responses_Ok);
```

Comment:
* When you say "all", it's quite unclear what it encompasses.
* When multiple settings fail to load, multiple dialogs will be shown, which would probably be annoying.
* I'd argue that settings not loading should not necessarily raise "FAIL FAIL". The settings might simply not be in the ini file and that would be okay.

So in the end, I'd probably always show a single dialog with a text akin to:
> Loading settings finished.
> Network: (Not) loaded
> Connect: (Not) loaded

Reply:
Edit: checked the code and settings load returns false if the config is somehow malformed or if the ini file is missing. So it is probably okay to call it  "failed".

Reply:
I've adapted to it, but used "+" and "-" instead, as the Dialog max msg length is rather limited and more messages will be added in the future.